src/frontend/src/instance/builder.rs

```diff
@@ -41,6 +43,7 @@ use crate::script::ScriptExecutor;
 /// The frontend [`Instance`] builder.
 pub struct FrontendBuilder {
     kv_backend: KvBackendRef,
+    cache_registry: CacheRegistryRef,
```

Comment:
Can we remove the `cache_invalidator` field, just use `cache_registry ` as invalidator?